nrf_security/cmake/symbol_rename.cmake

```diff
@@ -12,50 +12,108 @@
 # If two or more backends are enabled for a given function, each of them will
 # be renamed to be unique and a glue layer will be used as new front-end.
 #
+
+#
+# This macro comments out the instruction to rename a symbol
+# in case the configuration is not glued and enabled in backend
```

Comment:
I can't understand if this is saying,

the configuration is not (glued and enabled) in backend

or

the configuration is (not glued) and (enabled) in backend

please re-write to clarify

Reply:
Had a parenthesis combining (not glued and enabled), changed to enabled, but not glued

Reply:
Fixed